test/data/sandbox/ddt_test.ddt.js

```diff
@@ -1,18 +1,24 @@
+Feature('DDT');
```

Comment:
Previously user was be able to set datatable for Feature. This implementation doesn't allow to do this. Is this OK?